<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAttributeWidgetEditGroupBox</class>
 <widget class="QgsCollapsibleGroupBox" name="QgsAttributeWidgetEditGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gsCollapsibleGroupBox</string>
  </property>
  <property name="title">
   <string>Widget Display</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCheckBox" name="mRelationShowLinkCheckBox">
     <property name="text">
      <string>Show link button</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="mShowLabelCheckBox">
     <property name="text">
      <string>Show label</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="mRelationShowUnlinkCheckBox">
     <property name="text">
      <string>Show unlink button</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="mRelationShowSaveChildEditsCheckBox">
     <property name="text">
      <string>Show save child layer edits button</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
